opm/material/thermal/NullSolidEnergyLaw.hpp

```diff
@@ -50,7 +50,7 @@ class NullSolidEnergyLaw
      * This solid energy law simply returns 0.
      */
     template <class FluidState, class Evaluation = typename FluidState::Scalar>
-    static Evaluation solidInternalEnergy(const Params& params, const FluidState& fluidState)
+    static Evaluation solidInternalEnergy(const Params& params OPM_UNUSED, const FluidState&)
```

Comment:
that one was missed. I'll merge anyway if you want to avoid the jenkins dance...

Reply:
i don't get what you mean.

Reply:
`fluidState` parameter removed instead of marked unused

Reply:
sorry for being slow. updated.